pkg/proxy/accesslog/record.go

```diff
@@ -282,6 +282,18 @@ type LogRecordDNS struct {
 	// Empty or undefined may indicate older cilium versions, as it is expected
 	// to be filled in.
 	ObservationSource DNSDataSource `json:"ObservationSource,omitempty"`
+
+	// RCode is the response code
+	// defined as per https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-6
+	RCode int
+
+	// QTypes are question types in DNS message
+	// https://www.ietf.org/rfc/rfc1035.txt
+	QTypes []uint16
+
+	// AnswerTypes are record types in the answer section
+	// https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4
+	AnswerTypes []uint16
```

Comment:
In the comments for the new fields, could you include a one-line on how to convert the numbers to text? something like `dns.TypeToString[answerType]` for this one.

Reply:
I think `dns` package sports only `RcodeToString` map.